parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -744,7 +744,7 @@ configGPUDrivers() {
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         mkdir -p /opt/{actions,gpu}
         if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
-            ctr image pull $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG
+            ctr -n k8s.io image pull $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG
```

Comment:
now that you pull it under the k8s.io namespace, when you use the image to install the driver, do you need to reference that namespace ?

Reply:
Good call - I've updated the run commands as well. Surprisingly, the Agentbaker e2es for GPUs didn't fail before that. I wonder why.